CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+# [0.14.0] - 2022-01-05
+
+### Added
+
+- Web: Add removeLinks and getLinks method
+- Mobiles: Code block bindings
```

Comment:
Looks like it's the first time we are "changelogging" something for both mobile platforms. I wonder if we want to go with "Mobile", "Mobiles" or "iOS/Android" but I guess we should make a choice now and stick to it.

Reply:
I'm in favour `ios/Android`, what do you think about it ?

Reply:
`iOS/Android` or `iOS+Android`, maybe.

Reply:
I replaced `Mobiles` by `iOS/Android`